src/fastoad/notebooks/02_explanation/03_scipy.ipynb

```diff
@@ -0,0 +1,299 @@
+{
```

Comment:
<blockquote>The result is very close to the one obtained in the previous notebook.</blockquote><p>"Identical" will be fine, with the accuracy adjustment I suggest in notebook 2.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>